.github/workflows/ci_l2.yaml

```diff
@@ -1,14 +1,10 @@
-name: CI L2
+name: CI (L2)
 on:
   merge_group:
   pull_request:
     branches: ["**"]
-    paths-ignore:
-      - "README.md"
-      - "LICENSE"
-      - "**/README.md"
-      - "**/docs/**"
-      - "crates/vm/levm/**" # We ran this in a separate workflow
+    paths:
```

Comment:
since it's added to merge groups, we still have a safety change for l1-only changes